modules/message/src/main/JsonView.scala

```diff
@@ -0,0 +1,44 @@
+package lila.message
+
+import play.api.libs.json._
+import play.api.mvc.Result
+import play.api.mvc.Results._
```

Comment:
Couldn't tell which two you were pointing to, but it looks like these two were superfluous and I deleted them:

import scala.concurrent.duration._
import scala.concurrent.{ Future }